mobile/src/libs/components/loader/loader.tsx

```diff
@@ -5,7 +5,15 @@ import { AppColor } from '#libs/enums/enums';
 
 import { styles } from './styles';
 
-const Loader: React.FC = () => {
+type Properties = {
+  isVisible: boolean;
+};
+
+const Loader: React.FC<Properties> = ({ isVisible }) => {
+  if (!isVisible) {
+    return null;
+  }
```

Comment:
Loader has only one responsibility - to show some loading icon, not to check the condition whenether we need to show it